src/sanctum/src/PersonalAccessToken.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Hypervel\Sanctum;
+
+use Hyperf\Database\Model\Model;
```

Comment:
If we only plan to support Hypervel, we can use `Hypervel\Database\Eloquent\Model::class` for more model features.